librz/core/cautocmpl.c

```diff
@@ -271,6 +271,7 @@ static void autocmplt_cmd_arg_file(RzLineNSCompletionResult *res, const char *s,
 #endif
 		char *tmp = rz_str_newf(fmt, RZ_SYS_DIR, input);
 		if (!tmp) {
+			free(input);
 			return;
 		}
 		free(input);
```

Comment:
```suggestion
		char *tmp = rz_str_newf(fmt, RZ_SYS_DIR, input);
		free(input);
		if (!tmp) {
			return;
		}
```
input is always freed, so we can free it after `rz_str_newf`